deep_qa/models/reading_comprehension/bidirectional_attention.py

```diff
@@ -250,6 +250,14 @@ def score_instance(self, instance: CharacterSpanInstance):
 
     @staticmethod
     def get_best_span(span_begin_probs, span_end_probs):
+        if len(span_begin_probs.shape) > 2 or len(span_end_probs.shape) > 2:
```

Comment:
well there isn't really a `(x, 1)` case --- This check is here just to ensure that no higher-order input isn't passed in that we aren't expecting (since that would most likely be a bug somewhere else). My main rationale for separating them was to make it easier to see which things are failing from the traceback.